front-end/src/components/league-page/leaderboard/row.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { Box } from '@mui/system';
+import { getGradient } from '../general';
+import { Theme, Typography } from '@mui/material';
+import ImgBox from '../../risen-box/img-box';
+
+export interface LeaderboardRowProps {
+    gamesElement: React.ReactNode,
+    identifierElement: React.ReactNode,
+    mainValueElement: React.ReactNode,
+}
+
+export interface RowMainValue {
+  value: number,
+  formatter: (value: number) => string
+}
+export function Row(bgColor: string, nameTypography: React.ReactNode, gamesTypography: React.ReactNode, mainValueTypography: React.ReactNode) {
+  return (
+    <Box sx={{ pt: 0.5, pb: 0.5, display: 'flex', flexDirection: 'row', justifyContent:'space-between', background: getGradient(bgColor) }}>
```

Comment:
You can set these to be slightly transparent I think, so the parents background gradient will seep through. Instead of needing to manually set every gradient.

Reply:
Each row needs to manually set the gradiant the the bgColor is not the same. 

If you see this image, the rows are slightly off colored.
![image](https://github.com/afieldi/RisenStats/assets/23081610/5728d8a7-ebd1-44a7-bbb6-4d4532eb44d0)

Since each row is difference i need to calc the gradiant since its a diff color.

Reply:
Yeah ik. Every other row is the same as the background, then the others are darker. For the darker rows you set the background to a slightly transparent black, and it will work with the background to make the same effect.